pandas/core/indexes/multi.py

```diff
@@ -1798,13 +1798,26 @@ def to_flat_index(self):
 
         Examples
         --------
+
+        To convert a MultiIndex to an Index:
+
         >>> index = pd.MultiIndex.from_product(
         ...     [['foo', 'bar'], ['baz', 'qux']],
         ...     names=['a', 'b'])
-        >>> index.to_flat_index()
+        >>> index = index.to_flat_index()
+        >>> index
         Index([('foo', 'baz'), ('foo', 'qux'),
                ('bar', 'baz'), ('bar', 'qux')],
               dtype='object')
+
+        To convert an Index back to a MultiIndex:
```

Comment:
IMO this doesn't belong here - it's an example of `MultiIndex.from_tuples` constructor and not specifically to do with `to_flat_index`

Reply:
I'm getting some conflicting advice on this, so I'll let you decide amongst yourselves. I have no preference. 

My use case: I'm trying to merge two DataFrames, one uses a MultiIndex, the other uses an Index. The only way I found to get this to work is to collapse one MultiIndex and restore it later.

Reply:
@adamjstewart yeah we want an example of using *to_flat_index*, actually we have one above, so I guess the reference that you added is ok, but yeah let's kill this example as this is not relevant for this function.